openmetadata-ui/src/main/resources/ui/public/locales/en-US/Dashboard/QlikSense.md

```diff
@@ -108,6 +108,20 @@ You will have to replace new lines with `\n` and the final private key that you
 ```
 $$
 
+$$section
+### SSL CA $(id="caCertificate")
+The CA certificate used for SSL validation (`ssl_ca`)
```

Comment:
```suggestion
The CA certificate used for SSL validation.
```


The `ssl_ca` / `ssl_cert` / `ssl_key` are used for mysql and other sqlalchemy connectors, we can remove those for qliksense, tableau and kafka